model/src/main/scala/za/co/absa/atum/model/dto/AdditionalDataSubmitDTO.scala

```diff
@@ -0,0 +1,6 @@
+package za.co.absa.atum.model.dto
```

Comment:
licence is missing, please add it, the CI is failing

Reply:
Okay it will be added.

Reply:
still todo